src/authentication/UserManagerMock.js

```diff
@@ -0,0 +1,30 @@
+export class UserManagerMock {
+    settings;
+
+    constructor(settings) {
+        this.settings = settings;
+    }
+
+    getUser() {
+        return Promise.resolve(JSON.parse(sessionStorage.getItem("user")));
+    }
+
+    signinRedirect() {
+        window.location = "/sign-in-callback";
```

Comment:
This will not work when deployed, we don't host the app at '/'
You have to use the same code as in index.js
```
const base = document.querySelector('base');
const basename = base && base.href && new URL(base.href).pathname || process.env.PUBLIC_URL && new URL(process.env.PUBLIC_URL).pathname || '/';
```
please refactor index.js so that the computed basename is stored in a global variable and available anywhere in the application

Reply:
now that we have the base tag even in developpment, you can use window.location = "." it should work everywhere